app/scripts/modules/core/pipeline/config/stages/executionWindows/ExecutionWindowDayPickerComponent.html

```diff
@@ -0,0 +1,13 @@
+<div class="btn-group">
```

Comment:
File should be named `executionWindowDayPicker.component.html` to be consistent with other components.